src/recipes/custom-resources.md

```diff
@@ -0,0 +1,36 @@
+<!--
+  ~ Copyright (c) godot-rust; Bromeon and contributors.
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+# Custom Resources
+
+Custom `Resource`s are exposed for the end user to use within their development, but cannot provide any functionality automatically.
+For example, you can create a custom `AudioStream` type that handles a new and interesting audio file type.
```

Comment:
```suggestion
Custom `Resource`s are exposed to the end user to use within their development, but cannot provide any functionality automatically.
For example, you can create a custom `AudioStream` type that handles a new and interesting audio file type.
```

Also, can you reword the "cannot provide any functionality" part? It's not clear what this means. I would just describe the purpose of custom resources, not what they _can't_ do.

Reply:
I can do that. I'll make the commit change manually as well so this suggestion doesn't disappear for me